examples/tpm/tpm2_demo.c

```diff
@@ -85,14 +85,14 @@ static int TPM2_IoCb(TPM2_CTX* ctx, const byte* txBuf, byte* rxBuf,
 
     if (*spiDev == -1) {
         /* 33Mhz - PI has issue with 5-10Mhz on packets sized over 130 */
```

Comment:
Paul, would you mind updating the comment at line 87 about the speed? Otherwise this looks great!! Thanks

Reply:
David, sry the web gui from github is... getting used to you have a new Pull-Request